src/lib.rs

```diff
@@ -404,32 +259,4 @@ mod tests {
         assert_eq!(&digest[0], &0x90);
         assert_eq!(&mut digest[0], &mut 0x90);
     }
-
-    #[test]
-    fn overflow_count() {
```

Comment:
Why are you removing these two tests?

Reply:
Because they relied on the Context struct. Context discussion further down.